proto/desmos/subspaces/v1beta1/msgs.proto

```diff
@@ -0,0 +1,153 @@
+syntax = "proto3";
+package desmos.subspaces.v1beta1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/desmos-labs/desmos/x/staging/subspaces/types";
+
+// Msg defines subspaces Msg service.
+service Msg {
+
+  // CreateSubspace defines the method to create a subspace
+  rpc CreateSubspace(MsgCreateSubspace) returns (MsgCreateSubspaceResponse);
+
+  // TransferOwnership defines the method to transfer a subspace ownership
+  rpc TransferOwnership(MsgTransferOwnership) returns (MsgTransferOwnershipResponse);
```

Comment:
Instead of something like `MsgTransferOwnership`, I think it would be much better to implement a more genesis `MsgEditSubspace` that allows to edit the different things (except the `id`). This would also allow to change the name and the owner with a single message. We could then use the usual `[do-not-modify]` value to mean something shouldn't be edited, the same way we do with profiles.